### PR TITLE
Update various failure labels

### PR DESCRIPTION
**Pull request by ipa-hsd**
The PR is stacked behind https://github.com/robust-rosin/robust/pull/437

### CODE REVIEW

**Comment by gavanderhoorn** [comment]
Is this in a state it could be merged?

Some `.bug`s change to `SOFTWARE:NETWORK`, which I was just looking into. Right now there are only 5, 2 of which are `confidential` (and we typically don't list those).

Having a few more would be nice.

**Comment by ipa-hsd** [comment]
Here are few more: https://github.com/robust-rosin/robust/pull/428

File: care-o-bot/b826eae/b826eae.bug [hunk 1]
```diff
@@ -62,5 +62,5 @@ fault-codes:
   - BDO:ORCHESTRATION
   - PROGRAMMING:BROKEN-CONTRACT
 failure-codes:
-  - SOFTWARE:CRASHING
-  - SYSTEM:LOSS-OF-CONTROL
+  - SOFTWARE:BUILD
```
Comment (ChrisTimperley) [inline care-o-bot/b826eae/b826eae.bug:65]: Should be CRASHING

File: turtlebot/a4f35ee/a4f35ee.bug [hunk 2]
```diff
@@ -101,5 +101,5 @@ fix:
 fault-codes:
   - SYSTEMS:OS
 failure-codes:
-  - SOFTWARE:CRASHING
-  - SYSTEM:LOSS-OF-CONTROL
+  - SOFTWARE:BUILD
```
Comment (ChrisTimperley) [inline turtlebot/a4f35ee/a4f35ee.bug:104]: Let's change this back to CRASHING

File: turtlebot/a4f35ee/a4f35ee.bug [hunk 3]
```diff
@@ -101,5 +101,5 @@ fix:
...
+  - SYSTEM:NONE
```
Comment (ChrisTimperley) [inline turtlebot/a4f35ee/a4f35ee.bug:105]: After discussion at the meeting: Let's stick to "System: Loss of functionality."

File: care-o-bot/b826eae/b826eae.bug [hunk 4]
```diff
@@ -62,5 +62,5 @@ fault-codes:
...
+  - SYSTEM:NONE
```
Comment (ChrisTimperley) [inline care-o-bot/b826eae/b826eae.bug:66]: From the meeting: Let's go with "Loss of functionality"

**Review by ChrisTimperley — APPROVED** [review]

**Pull request merged**